concent_api/core/views.py

```diff
@@ -419,6 +525,15 @@ def store_message(golem_message_type, task_id, raw_golem_message):
     return (golem_message, message_timestamp)
 
 
+def store_message_auth(golem_message, provider_public_key, requestor_public_key):
```

Comment:
The function tries to insert keys into fields that are non-nullable so it should state as a precondition that `provider_public_key` and `requestor_public_key` are not `None` (using `assert`).

Reply:
:+1: